hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/validate/HazelcastSqlOperatorTable.java

```diff
@@ -337,6 +341,59 @@
 
     //#endregion
 
+    //#region String functions
+
+    public static final SqlBinaryOperator CONCAT = new SqlBinaryOperator(
+        "||",
+        SqlKind.OTHER,
+        60,
+        true,
+        ReturnTypes.DYADIC_STRING_SUM_PRECISION_NULLABLE,
+        HazelcastInferTypes.VARCHAR_IF_UNKNOWN,
+        OperandTypes.STRING_SAME_SAME
```

Comment:
notAny

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/17379/commits/6a01e16bd9b3450ef82536e461db30bb9017f1bc